packages/teams-js/test/private/hostEntity.spec.ts

```diff
@@ -96,7 +96,26 @@ describe('hostEntity', () => {
           const promise = hostEntity.tab.addAndConfigure(mockHostEntity);
           const message = utils.findMessageByFunc('hostEntity.tab.addAndConfigure');
           expect(message).not.toBeNull();
-          expect(message?.args).toEqual([mockHostEntity, null]);
+          expect(message?.args).toEqual([mockHostEntity, null, null]);
```

Comment:
Can you pass appTypes here to make sure that these arguments are in the right order?